src/drivers/QtTestDriver.cpp

```diff
@@ -0,0 +1,25 @@
+#include "cucumber-cpp/internal/drivers/QtTestDriver.hpp"
+#include "cucumber-cpp/internal/utils/qtCapture.hpp"
+
+#include <QtTest/QtTest>
+
+namespace cucumber {
+namespace internal {
+
+const InvokeResult QtTestStep::invokeStepBody() {
+    QtTestObject testObject(this);
+
+    qtCapture::Init();
+    qtCapture::BeginCapture();
```

Comment:
I'm assuming you're capturing stdout/stderr here because Qt tends to be overly verbose during execution? Is it possible to make Qt write to some alternate log sink instead?